### PR TITLE
Add git pod as an online tool

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: .gitpod.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM gitpod/workspace-full
+
+ENV PATH /home/linuxbrew/.linuxbrew/bin:$PATH
+
+# More information: https://www.gitpod.io/docs/config-docker/
+RUN sudo rm -rf /usr/bin/hd && \
+    brew install linuxsuren/linuxsuren/hd && \
+    hd install cli/cli
```
Comment (JohnNiang) [inline .gitpod.Dockerfile:8]: It seems that github cli isn't installed correctly.

```bash
gitpod /home/linuxbrew/.linuxbrew $ whereis hd
hd: /usr/bin/hd /usr/share/man/man1/hd.1.gz
gitpod /home/linuxbrew/.linuxbrew $ whereis gh
gh:
```

Comment (LinuxSuRen) [reply .gitpod.Dockerfile:8]: It works for me. You an visit [this link](http://gitpod.io/#/github.com/linuxsuren/devops-agent/tree/gitpod).

image

File: .gitpod.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM gitpod/workspace-full
+
+ENV PATH /home/linuxbrew/.linuxbrew/bin:$PATH
```
Comment (JohnNiang) [inline .gitpod.Dockerfile:3]: IMO, there is no need to explicitly append brew bin path. See here: <https://github.com/gitpod-io/workspace-images/blob/97e963a6b7d91ef2eb6e03d65f2de488297865a5/full/Dockerfile#L68>.

File: .gitpod.Dockerfile (hunk 1 above)
Comment (JohnNiang) [reply .gitpod.Dockerfile:8]: Sorry for that. I went into <https://gitpod.io/#https://github.com/kubesphere/devops-agent> incorrectly.

File: .gitpod.Dockerfile (hunk 2 above)
Comment (LinuxSuRen) [reply .gitpod.Dockerfile]: Yes, cool.

**Pull request merged**